weblogic/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,47 @@
+name: WebLogic
+files:
+- name: weblogic.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/jmx
+  - template: instances
+    options:
+    - template: instances/jmx
+      overrides:
+        host.value.example: localhost
+        port.value.example: 9090
+  - template: logs
+    example:
+    - type: file
+      path: <DOMAIN_DIR>/servers/<ADMIN_SERVER_NAME>/logs/<ADMIN_SERVER_NAME>.log
+      source: weblogic
+      service: admin-server
```

Comment:
We rarely set `service`, do we need to do so here?

Reply:
We discussed it in this conversation: https://github.com/DataDog/integrations-core/pull/10559#discussion_r762051463 

The server (admin server, managed server) and domain logs are located in different directories, but they share the same multi_line log processing rule. HTTP access logs have a different multi_line log processing rule. 

With that in mind, should I remove `service` altogether and have examples of a server log and http-access log? i.e.

```
    - type: file
      path: <DOMAIN_DIR>/servers/<SERVER_NAME>/logs/<SERVER_NAME_OR_DOMAIN_NAME>.log
      source: weblogic
      log_processing_rules:
        - type: multi_line
          name: new_log_start_with_date
          pattern: (\####)?<\w{3} (0?[1-9]|[12][0-9]|3[01]), \d{4}
    - type: file
      path: <DOMAIN_DIR>/servers/*/logs/access.log
      source: weblogic
      log_processing_rules:
        - type: multi_line
          name: new_log_start_with_date
          pattern: .*\[\d{2}\/(\w{3}|\w{4})\/\d{4}:\d{2}:\d{2}:\d{2} 
```
cc @ChristineTChen

Reply:
Nah, it's fine